test/parallel/test-tls-connect-given-socket.js

```diff
@@ -23,16 +23,16 @@
 const common = require('../common');
 if (!common.hasCrypto)
   common.skip('missing crypto');
+const fixtures = require('../common/fixtures');
 
 const assert = require('assert');
 const tls = require('tls');
 const net = require('net');
 const fs = require('fs');
-const path = require('path');
 
 const options = {
-  key: fs.readFileSync(path.join(common.fixturesDir, 'test_key.pem')),
-  cert: fs.readFileSync(path.join(common.fixturesDir, 'test_cert.pem'))
+  key: fs.readFileSync(fixtures.path('test_key.pem')),
```

Comment:
Sorry, was reading too fast!

This could be cleaned up further by making use of [`fixtures.readSync()`](https://github.com/nodejs/node/tree/master/test/common#fixturesreadsyncargs-enc) to replace the `fs.readFileSuync` _and_ the `path.join`!

Something like `key: fixtures.readSync('test_key.pem'),`

Reply:
I will fix this right away.